spec/features/admin/leads_spec.rb

```diff
@@ -45,7 +45,7 @@
     login(admin: true)
     user = create(:user, email: 'lead@example.com', name: 'New Lead')
     role = create(:volunteer_role, name: 'Role1')
-    create(:volunteer, user: user, volunteer_role: role, lead: true)
+    create(:volunteer, user:, volunteer_role: role, lead: true)
```

Comment:
And this

Reply:
So this collection of changes was actually a `rubocop` thing. `Style/HashSyntax: Omit the hash value` specifically as the key is behind the scenes it'll automagically change this to `{ user: myuserdatahere# }`. We can force it to not do this by changing `EnforcedShorthandSyntax` to be `never`

Reply:
OMG well I learned a thing about ruby today. That is mad. Do I like it? I'm not sure. But I repeal my comment.